.gitignore

```diff
@@ -91,3 +91,6 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
+#vscode
+.vscode
```

Comment:
Could you use an editor configuration to avoid these line ending issues in the future. Such issues will lead to PRs not being merged. So fix these first. Otherwise, good work.